docker-compose.yml

```
@@ -108,19 +133,25 @@ services:
     environment:
       - ELASTIC_PASSWORD=serxdfcghjfc
     ports: [ 8311:8311 ]
+    networks: [ datalayer-elasticsearch ]
```

The datalayer proxy isn't set up yet so search needs to be able to connect directly to elasticsearch for now
```suggestion
    networks: [ datalayer-elasticsearch, search-datalayer ]
```